src/RoslynPad.Hosting/ExecutionHost.cs

```diff
@@ -156,14 +159,18 @@ private static DelegatingTextWriter CreateConsoleWriter()
 
         public ExecutionHost(string hostPath, string initialWorkingDirectory,
```

Comment:
I think it might be better to create a class for all of these parameters instead of copying them down the call chain. Note that it would need to be a `[DataContract]` in order to pass via WCF to the `Initialize` method.